### DISCUSSION TITLE
Twitter/x logo does not appear on the About page after adding the x_username

### DISCUSSION THREAD

**Discussion by dua-arpit** [alshedivat/al-folio#2616] [Q&A] [answered]
Hi, I saw that there is an option to add the x_username. I added that, but I don't see that doing anything on the About page. Is the functionality for the Twitter logo/link to appear at the bottom of the About page not already built in? I thought I'd ask before tweaking anything 

I also checked social.liquid. It already has the lines 

```
{% if site.x_username %}
  <a href="https://twitter.com/{{ site.x_username }}" title="X"><i class="fa-brands fa-x-twitter"></i></a>
{% endif %}
```

**Comment by george-gca** [comment]
Does this happens locally or on the online version? If this happens with the online version, you have to wait for the build action to finish in your GitHub repo.

**Reply by dua-arpit** [reply]
On the online version. I did wait for the build actions to finish.

**Reply by george-gca** [reply]
Are you using the latest version of our code? Can you share your code?
Also, try to access your site on a incognito window.

**Reply by dua-arpit** [reply]
Yes, it seems to be the latest version. I tried the incognito window and it does not show.  

For the code, I only defined the variable x_username in _config.yml. I see the following lines in social.liquid, which uses the x_username variable. 

```
{% if site.x_username %}
  <a href="https://twitter.com/{{ site.x_username }}" title="X"><i class="fa-brands fa-x-twitter"></i></a>
{% endif %}
```

Do you want me to share the complete code?

**Reply by george-gca** [reply]
Just tested the latest version of our code by adding this:

```yml
x_username: HIDEO_KOJIMA_EN # your X handle
```

Which got me this:

![image](https://github.com/user-attachments/assets/f0497047-fee3-4aa3-ba6c-cc9ff65209e8)

Maybe you did some other changes that affected this behavior.

**Reply by george-gca** [reply, marked as answer]
Just checked your code. You should put your user before the [comments](https://spacelift.io/blog/yaml-comments) on the line, meaning before the `#` and after the `:`. Everything on the lines after the # are comments and are ignored by the build.

**Reply by dua-arpit** [reply]
oh goodness, thanks
